tests/test_scripts.py

```diff
@@ -65,9 +65,32 @@ def test_main_console(script_mod):
         script_mod.main_console()
 
 
-def test_train_preference_comparisons_main(tmpdir):
+PREFERENCE_COMPARISON_CONFIGS = [
+    {},
+    {
+        "trajectory_path": "tests/testdata/expert_models/cartpole_0/rollouts/final.pkl",
+    },
+    {
+        "agent_path": (
+            "tests/testdata/expert_models/cartpole_0/policies/final/model.zip"
+        ),
+        # TODO(ejnnr): the policy we load was trained on 8 parallel environments
+        # and for some reason using it breaks if we use just 1 (like would be the
+        # default with the fast named_config)
+        "num_vec": 8,
+        # we're putting this here to test it at some point without having
```

Comment:
"this", "it" -- can you be a bit more precise? Even if just "the below" (e.g. right now I'm not sure if you're referring to the immediate next line or the next two lines).

Reply:
I meant both of them, I've rewritten the comment